config/i18n/locales/swahili/translations.json

```diff
@@ -1,67 +1,67 @@
 {
   "buttons": {
-    "forum": "Forum",
-    "donate": "Donate",
-    "load-more-articles": "Load More Articles"
+    "forum": "Jukwaa",
+    "donate": "Changia",
+    "load-more-articles": "Onyesha Makala Zaidi"
   },
   "search": {
-    "label": "Search",
-    "accessible-name": "Submit your search query",
-    "placeholder": "Search 8,000+ tutorials",
-    "see-results": "See all results for {{ searchQuery }}",
-    "no-tutorials": "No tutorials found"
+    "label": "Tafuta",
+    "accessible-name": "Wasilisha swali lako la utafutaji",
+    "placeholder": "Tafuta kati ya mafunzo zaidi ya 8,000+",
```

Comment:
Thanks for translating this. But could we adapt it to something more general without a number?

In English it reads "Search 8,000+ tutorials" because there are more than 8,000 published articles. But because there are less than 100 published tutorials on the Swahili publication, perhaps it could read "Search our tutorials".